src/pages/research.jsx

```diff
@@ -0,0 +1,257 @@
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from 'react';
+import { graphql } from 'gatsby';
+import { navigate } from '@reach/router';
+
+import Layout from '../components/layout';
+import HeaderSearchArea from '../components/HeaderSearchArea';
+import Pager from '../components/Pager';
+import ArticlePreview from '../components/ArticlePreview';
+import ArticlePreviewList from '../components/ArticlePreviewList';
+
+import { contentPage, pager } from '../styles/contentPage.module.scss';
+
+import {
+  instrumentSection,
+  topicsMenuHeader,
+  instruments,
+  searchArea,
+  hidden,
+  accordionMenu,
+  menuLogo,
+} from '../styles/researchPage.module.scss';
+
+import utils from '../utils';
+
+const ResearchInstrumentList = ({ instrumentData }) => (
+  <ul className={instruments} aria-label="List of research instruments">
+    {instrumentData.map((edge) => (
+      <li key={edge.node.fields.slug}>
+        <ArticlePreview
+          title={edge.node.frontmatter.title}
+          imageAlt={edge.node.frontmatter.imageAlt}
+          previewImage={edge.node.frontmatter.featuredImage}
+          articleUrl={edge.node.fields.slug}
+        />
+      </li>
+    ))}
+  </ul>
+);
+
+const Plus = () => <div className={menuLogo}>+</div>;
+const Minus = () => <div className={menuLogo}>-</div>;
+
+const AccordionMenuHeader = ({ children, handleClick, isOpen }) => (
+  <button
+    type="button"
+    onClick={handleClick}
+    className={accordionMenu}
+    aria-label={isOpen ? 'Close the menu' : 'Open the menu'}
```

Comment:
This would prolly work best if you removed the aria-label, made the whole button <h2> (instead of h2 being inside it), though you'd have to update styles too, and added an `aria-expanded`-section there. The plus and minus-components should also have `aria-hidden="true"` as they are only cosmetic. In addition these require some kind of hover state.

more inspiration in here: 
https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html